liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -192,16 +192,16 @@ public class DevMojo extends LooseAppSupport {
     protected boolean pollingTest;
 
     /**
-     * Dockerfile used to build a Docker image to then start a container with
+     * Containerfile used to build an image to then start a container with
      */
-    @Parameter(property = "dockerfile")
-    private File dockerfile;
+    @Parameter(alias="dockerfile", property = "containerfile")
+    private File containerfile;
```

Comment:
So when we aliased some parameters back in 3.0 timeframe, both the alias and property were the new value, and the variable was the old value:
```
    @Parameter(alias="runtimeArchive", property = "runtimeArchive")
    protected File assemblyArchive;
```

Did you test using both values to see if this works as you expected?